src/tools/tidy/src/deps.rs

```diff
@@ -1,6 +1,6 @@
 //! Checks the licenses of third-party dependencies by inspecting vendors.
 
-use std::collections::{BTreeSet, HashMap, HashSet};
+use std::collections::{BTreeSet, HashSet, HashMap};
```

Comment:
Restore the order of these. I believe rustfmt will complain about misordered imports.